shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -63,8 +63,6 @@ void FlutterTizenView::SetEngine(std::unique_ptr<FlutterTizenEngine> engine) {
   text_input_channel_ = std::make_unique<TextInputChannel>(
       internal_plugin_registrar_->messenger(),
       std::make_unique<TizenInputMethodContext>(window_.get()));
-
-  OnRotate(window_->GetRotation());
```

Comment:
Could you check if the rotation works correctly without this line when an app is **started** in **landscape** mode on **mobile**?

Reply:
@swift-kim Of course :) I checked and it works fine.